test/unit/tool_util/mulled/test_get_tests.py

```diff
@@ -72,9 +72,11 @@ def test_get_test_from_anaconda():
 
     # test for package defining tests in info/recipe/run_test.sh
     tests = get_test_from_anaconda(
-        "https://anaconda.org/bioconda/ucsc-pslmap/366/download/linux-64/ucsc-pslmap-366-hdd26221_0.tar.bz2"
+        "https://anaconda.org/bioconda/mugsy/1.2.3/download/noarch/mugsy-1.2.3-hdfd78af_4.tar.bz2"
     )
-    assert tests and tests["commands"] == ['#!/bin/bash && pslMap 2> /dev/null || [[ "$?" == 255 ]] && ']
+    assert tests and tests["commands"] == [
+        "#!/bin/bash &&  && export MUGSY_INSTALL=${PREFIX}/bin && mugsy -h | grep mugsy > /dev/null && mugsyWGA  --version && synchain-mugsy 2>&1 | grep mugsy > /dev/null && "
```

Comment:
This is clearly broken, I don't think we should even try to translate a `run_test.sh` into a one-line command.